api/src/test/kotlin/edu/wgu/osmt/collection/CollectionRepositoryTest.kt

```diff
@@ -189,4 +191,28 @@ class CollectionRepositoryTest: SpringTest(), BaseDockerizedTest, HasDatabaseRes
         assertThat(batchResult?.modifiedCount).isEqualTo(skillCount*3)
     }
 
+    @Test
+    fun `remove finds and succesfully removes an existing collection`() {
+        // Arrange
+        val collection = collectionRepository.create(UUID.randomUUID().toString(), userString)!!.toModel()
+
```

Comment:
Can we please add some RSDs to test the delete from the relationship table?

Reply:
added assertion for RSD table changes